src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -25,6 +25,24 @@ class Rule:
     dst_schema: str
     src_table: str
     dst_table: str
+    skip_table: bool = False
+
+    def __init__(
+            self,
+            workspace_name: str,
+            catalog_name: str,
+            src_schema: str,
+            dst_schema: str,
+            src_table: str,
+            dst_table: str,
+            skip_table: str | bool = False):
+        self.workspace_name = workspace_name
+        self.catalog_name = catalog_name
+        self.src_schema = src_schema
+        self.dst_schema = dst_schema
+        self.src_table = src_table
+        self.dst_table = dst_table
+        self.skip_table = str(skip_table).lower() == "true"
```

Comment:
Why do we need this?.. remove.

Reply:
I would like to support converting string to boolean when reading from the CSV. Or should I move the conversion to the load method.

Reply:
Removed